### ISSUE TITLE
Does Janus support multiprocessing.Queue?

### ISSUE THREAD

**Issue by HWiese1980** [aio-libs/janus#198]
I make heavy use of Janus Queue in one of my projects to asynchronize thread communication. Now I would like to do the same with communication with subprocesses. Does Janus Queue provide the same level of compatibility for `multiprocessing.Queue` as it does for `queue.Queue`?

Cheers

**Comment by asvetlov** [comment]
No. Do you want to propose a pull request with requested functionality?

**Comment by HWiese1980** [comment]
Would be an option... I don't have any code to contribute yet though and I'm currently too busy to write it.

**Comment by ajslater** [comment]
I believe [aioprocessing](https://github.com/dano/aioprocessing ) attempts to address this issue.
